backends/ldap/auth.go

```diff
@@ -145,6 +146,16 @@ func constructor(config string) (backend.Backend, error) {
 		}
 	}
 
+	if s, found := options["authenticated"]; found {
+		if s == "true" {
```

Comment:
Please look at https://golang.org/pkg/strconv/#ParseBool

This whole if else if else block could be

```golang
var err error
us.authenticated, err = strconv.ParseBool(s)
If err != nill {
    return nil, fmt.Errorf("unable to parse authenticated %q: %v", s, err)
}
```